src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Gateway/Pages/OrganisationRisk.cshtml

```diff
@@ -6,22 +6,20 @@
     Layout = "_Layout";
 }
 
-@{await Html.RenderPartialAsync("~/Views/Roatp/Apply/Gateway/Pages/_GatewayAssessmentBreadcrumb.cshtml"); }
+@{ await Html.RenderPartialAsync("~/Views/Roatp/Apply/Gateway/Pages/_GatewayAssessmentBreadcrumb.cshtml"); }
```

Comment:
Use of deprecated HTML Helper functions. Use tag helpers

Reply:
done